backend/StreamServerApp/urls.py

```diff
@@ -3,10 +3,17 @@
 
 from . import views
 
+video_list = views.VideoViewSet.as_view({
+    'get': 'list',
+})
+
+video_detail = views.VideoViewSet.as_view({
+    'get': 'retrieve',
+})
+
```

Comment:
I am not sure this is the way to go.
DRF provides some router objects, see [here](https://www.django-rest-framework.org/api-guide/viewsets/#example).
I am fine with this way, but maybe when adding more views (e.g. for Collections), it will make it easier.